modules/apps/frontend-js/frontend-js-components-web/src/main/resources/META-INF/resources/forms/common/FieldFeedback.tsx

```diff
@@ -19,13 +19,17 @@ export default function FieldFeedback({
 
 	return (
 		<ClayForm.FeedbackGroup className="field-feedback" {...otherProps}>
+			{helpMessage && (
+				<ClayForm.FeedbackGroup>
+					<ClayForm.HelpText>{helpMessage}</ClayForm.HelpText>
```

Comment:
`HelpText` should be placed at the bottom of a `FeedbackGroup`.

Reply:
@ethib137 it looks like I was wrong, design changed the order https://www.figma.com/design/YNNkt9Xd6ImDtEvIz4tETF/Lexicon-Components?node-id=1-12&p=f&t=BcMj0T2emxyZOzi4-0.

Edit: It also makes sense for the error / state messages to come after so it doesn't cause text to jump.